addons/web/static/src/core/utils/sortable_list.js

```diff
@@ -229,7 +231,7 @@ export const useSortableList = makeDraggableHook({
          */
         if (ctx.nest) {
             const xInterval = ctx.prevNestX - ctx.pointer.x;
-            if (ctx.nestInterval - (-1) ** ctx.isRTL * xInterval < 0) {
+            if (ctx.nestInterval - (-1) ** ctx.isRTL * xInterval < 1) {
```

Comment:
Why is it now 1 instead of 0 ? What essentially changed from the Knowledge version for this ?